frontend/wallet/src/render/views/sidebar/UnlockModal.re

```diff
@@ -13,6 +21,20 @@ module UnlockMutation = ReasonApollo.CreateMutation(UnlockAccount);
 type modalState = {error: option(string)};
 [@react.component]
 let make = (~account, ~onClose, ~onSuccess) => {
+  let intl = useIntl();
+
+  // TODO This string needs to be titlecased
+  // Currently cannot style modal title
+  // Larger discussion is required to settle on how translations are capitalized
```

Comment:
So here is where localization gets annoying because it breaks my pretty lowercased json pattern. The intl'ed message should be titlecased. But...I've lowercased it in the json. simple fix would just be to write the translation titlecased. 🤷‍♂

Reply:
For the title of the modal I'd imagine we can just make it always titlecase (unless there's a case I'm missing @figitaki)
Either way I don't think you need to be blocked on this particular issue.